index.html

```diff
@@ -5,8 +5,60 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Weather Report</title>
+    <link href="styles/index.css" rel="stylesheet">
+    <script src="https://cdn.jsdelivr.net/npm/axios/dist/axios.min.js"></script>
+    <script src="src/index.js"></script> 
+    <!-- <script src="./node_modules/axios/dist/axios.min.js"></script> -->
```

Comment:
Here we should be using our local version of axios - the one on line 11. It may not have been working for you if you forgot to do a `yarn install`.